src/Redux/configureStore.js

```diff
@@ -0,0 +1,12 @@
+import { configureStore } from '@reduxjs/toolkit';
+import bookReducer from './book/book';
+import categoriesReducer from './categories/categories';
+
+const store = configureStore({
```

Comment:
Uses `configureStore` to create a reducer 🔥

Reply:
Thanks a lot, I completely forgot and acctually didn't pay attention to that part. Thanks for pointing that out.